bigbluebutton-html5/imports/ui/components/external-video-player/modal/component.jsx

```diff
@@ -0,0 +1,147 @@
+import React, {Component} from "react";
+import { withModalMounter } from '/imports/ui/components/modal/service';
+
+import ModalBase from '/imports/ui/components/modal/base/component';
+import Button from '/imports/ui/components/button/component';
+
+import {styles} from './styles';
+
+import { defineMessages, injectIntl } from 'react-intl';
+
+const intlMessages = defineMessages({
+  start: {
+    id: 'app.externalVideo.start',
+    description: 'Share youtube video',
+  },
+  stop: {
+    id: 'app.externalVideo.stop',
+    description: 'Stop sharing video',
+  },
+  urlError: {
+    id: 'app.externalVideo.urlError',
+    description: 'Not a video URL error',
+  },
+  input: {
+    id: 'app.externalVideo.input',
+    description: 'Video URL',
+  },
+  title: {
+    id: 'app.externalVideo.title',
+    description: 'Modal title',
+  },
+  close: {
+    id: 'app.externalVideo.close',
+    description: 'Close',
+  },
+});
+
+const YOUTUBE_PREFIX = "https://youtube.com/watch?v=";
+
+const getUrlFromVideoId = (url) => {
+  return url ? `${YOUTUBE_PREFIX}${url}` : '';
+}
+
+class ExternalVideoModal extends Component {
+
+  state = {
+    url: getUrlFromVideoId(this.props.url),
```

Comment:
Same - `this.props.videoId`. Also it would be nice if you could make the linter a bit happier by adding
`const { x } = this.props;`
`const { y } = this.state;`, etc - check the linter report either locally or when it's generated on TravisCI after pushing a commit